src/components/newProposal/steps/Voting.tsx

```diff
@@ -70,7 +67,8 @@ export type EndTimeType = 'duration' | 'end-custom';
 export const Voting = () => {
   const { setStep, dataStep2, setDataStep2 } = useNewProposalFormContext();
 
-  const { settings, error } = useVotingSettings();
+  const { data: minDurationBN, error } = usePartialVotingProposalMinDuration();
+  const minDuration = minDurationBN?.toNumber() ?? null;
```

Comment:
Same as I mentioned above. Maybe we ought to actually catch possible errors with the `toNumber()` conversion